wordpress-mysql-replication/nested/mysql-replication.json

```diff
@@ -1,551 +1,548 @@
+         }
+      },
+      "virtualNetworkName":{  
+         "type":"string",
+         "defaultValue":"mysqlvnet",
+         "metadata":{  
+            "description":"New or existing Virtual network name"
+         }
+      },
+      "dbSubnetName":{  
+         "type":"string",
+         "defaultValue":"default",
+         "metadata":{  
+            "description":"subnet name for the MySQL nodes"
+         }
+      },
+      "vnetAddressPrefix":{  
+         "type":"string",
+         "defaultValue":"10.0.0.0/16",
+         "metadata":{  
+            "description":"IP address in CIDR for virtual network"
+         }
+      },
+      "dbSubnetAddressPrefix":{  
+         "type":"string",
+         "defaultValue":"10.0.1.0/24",
+         "metadata":{  
+            "description":"IP address in CIDR for db subnet"
+         }
+      },
+      "dbSubnetStartAddress":{  
+         "type":"string",
+         "defaultValue":"10.0.1.4",
+         "metadata":{  
+            "description":"Start IP address in the subnet for the VMs"
+         }
+      },
+      "imagePublisher":{  
+         "type":"string",
+         "defaultValue":"OpenLogic",
+         "allowedValues":[  
+            "OpenLogic"
+         ],
+         "metadata":{  
+            "description":"publisher for the VM OS image"
+         }
+      },
+      "imageOffer":{  
+         "type":"string",
+         "defaultValue":"CentOS",
+         "allowedValues":[  
+            "CentOS"
+         ],
+         "metadata":{  
+            "description":"VM OS name"
+         }
+      },
+      "imageSKU":{  
+         "type":"string",
+         "defaultValue":"6.6",
+         "allowedValues":[  
+            "6.5",
+            "6.6"
+         ],
+         "metadata":{  
+            "description":"VM OS version"
+         }
+      },
+      "mysqlFrontEndPort0":{  
+         "type":"int",
+         "defaultValue":3306,
+         "metadata":{  
+            "description":"MySQL public port master"
+         }
+      },
+      "mysqlFrontEndPort1":{  
+         "type":"int",
+         "defaultValue":3307,
+         "metadata":{  
+            "description":"MySQL public port slave"
+         }
+      },
+      "sshNatRuleFrontEndPort0":{  
+         "type":"int",
+         "defaultValue":64001,
+         "metadata":{  
+            "description":"public ssh port for VM1"
+         }
+      },
+      "sshNatRuleFrontEndPort1":{  
+         "type":"int",
+         "defaultValue":64002,
+         "metadata":{  
+            "description":"public ssh port for VM2"
+         }
+      },
+      "mysqlProbePort0":{  
+         "type":"int",
+         "defaultValue":9200,
+         "metadata":{  
+            "description":"MySQL public port master"
+         }
+      },
+      "mysqlProbePort1":{  
+         "type":"int",
+         "defaultValue":9201,
+         "metadata":{  
+            "description":"MySQL public port slave"
+         }
+      },
+      "artifactsPath":{  
```

Comment:
See https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc for using artifacts the sample